### PR TITLE
Modifications to Chapters 3 and 4 of the Users' Guide

### PR DESCRIPTION
**Pull request by JeffBeck-NOAA**
## DESCRIPTION OF CHANGES: 
Changes to Chapters 3 ahd 4 of the Users' Guide: remove placeholders, update text for recent changes, modify syntax, and remove references to GFDLgrid.

## TESTS CONDUCTED: 
None.

### CODE REVIEW

File: docs/UsersGuide/source/SRWAppOverview.rst [hunk 1]
```diff
@@ -54,60 +53,59 @@ The cloned repository contains the configuration files and sub-directories shown
...
+Before building the executables, the build environment must be set up for your specific platform.
+Instructions for loading the proper modules and/or setting the correct environment variables 
+can be found in the ``docs/`` directory in files named ``build_<platform>_<compiler>.env.`` For the
+most part, the commands in those files can be directly copied and pasted, but you may need to modify
+certain variables such as the path to NCEP libraries for your specific platform.  Here is a directory
+listing example of these kinds of files: 
 
 .. code-block:: console
 
    $ ls -l docs/
```
Comment (climbfuji) [inline docs/UsersGuide/source/SRWAppOverview.rst:102]: This is in `env`, not in `docs`, correct?

Comment (JeffBeck-NOAA) [reply docs/UsersGuide/source/SRWAppOverview.rst:102]: Correct!  Thanks for catching that.

Comment (climbfuji) [reply docs/UsersGuide/source/SRWAppOverview.rst:102]: Line 103 in docs/UsersGuide/source/SRWAppOverview.rst is still `docs` (sorry if you are still working on it)

Comment (JeffBeck-NOAA) [reply docs/UsersGuide/source/SRWAppOverview.rst]: Thanks, fixed that one too!

**Review by JulieSchramm — APPROVED** [review]
Nice cleanup of Chapter 4.

**Pull request merged**